doc/rst/usingchapel/chplenv.rst

```diff
@@ -655,7 +655,14 @@ CHPL_UNWIND
        ========= =======================================================
 
    If unset, ``CHPL_UNWIND`` defaults to ``none``
-
+ 
+CHPL_RT_UNWIND 
+~~~~~~~~~~~~~~
+   Optionally, the ``CHPL_RT_UNWIND`` environment variable can be used to squash 
```

Comment:
Now you've forgotten to list the valid values in the description: `0`, `false`, or `no`.

Maybe try: 

```
.. tip::

    The ``CHPL_RT_UNWIND`` environment variable can be set to 
    ``0``, ``false``, or ``no`` to squash the output ...
```